hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -2982,6 +2982,16 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.datanode.fsdatasetasyncdisk.max.threads.per.volume</name>
+  <value>4</value>
+  <description>
+    The maximum number of threads per volume used to delete blocks on
```

Comment:
Maybe this could be used for other asynchronous operations in the future, not just deleting blocks, what do you think?

Reply:
Thanks @tomscut for your comments and review.
I agree to your suggestion. But now FsDatasetAsyncDiskService mainly aimed at removing block.
Do you have a better description of this property?

Reply:
I think it could be changed to this: `delete blocks` -> `process async disk operations`.